src/G2/Equiv/Tactics.hs

```diff
@@ -13,6 +13,7 @@ module G2.Equiv.Tactics
     , tryEquality
     , moreRestrictiveEqual
     , tryCoinduction
+    , tryCoinductionAll
```

Comment:
It's not clear to me that there's a ton of rhyme or reason to this export list anyway, but is there a reason we added this?